scaleway/helpers_rdb.go

```diff
@@ -116,16 +116,21 @@ func expandPrivateNetwork(data interface{}, exist bool) ([]*rdb.EndpointSpec, er
 	var res []*rdb.EndpointSpec
 	for _, pn := range data.([]interface{}) {
 		r := pn.(map[string]interface{})
-		ip, err := expandIPNet(r["ip_net"].(string))
-		if err != nil {
-			return res, err
-		}
+		ipNet := r["ip_net"].(string)
 		spec := &rdb.EndpointSpec{
 			PrivateNetwork: &rdb.EndpointSpecPrivateNetwork{
 				PrivateNetworkID: expandID(r["pn_id"].(string)),
-				ServiceIP:        &ip,
 			},
 		}
+		if len(ipNet) > 0 {
```

Comment:
```suggestion
		if ipNet != "" {
```
As ipNet is a string, check for zero-value